src/code/code_800EC960.c

```diff
@@ -3774,41 +3785,41 @@ void func_800F6828(u8 arg0) {
     }
 }
 
-void func_800F68BC(s8 arg0) {
-    D_8013060C = arg0;
+void Audio_SetDemoActive(s8 active) {
+    sAudioDemoActive = active;
 }
```

Comment:
even though they originally called cutscenes "demos" we generally try to avoid it when possible.
is this only used for scripted cutscenes? (ie those controlled by z_demo.c), or is it "cutscene" in a broader sense? there are a few different kind of cutscenes in the game

Reply:
Only used from z_demo, yes. Would Audio_SetCutsceneActive be the right name for that, or do we want something more specific?

Reply:
"Cutscene" sounds better but I'm not so sure about the "Active" part, I would just use generic "Flag" like `SetCutsceneFlag`, like what the debug string has (`DEMO FLAG`) and wait for further documentation to better understand the contexts where that function is used

Reply:
that sounds fine for now. my main concern is distinguishing a scripted cutscene from stuff like one point demos or player cutscenes, but I think anything labeled as a generic cutscene currently is assumed to mean the script system (ie CutsceneContext and related things)